app/src/main/java/org/mozilla/focus/utils/StatusBarUtils.java

```diff
@@ -0,0 +1,31 @@
+package org.mozilla.focus.utils;
+
+import android.view.View;
+import android.view.WindowInsets;
+
+/**
+ * Created by Fer on 08/03/2018.
+ */
+
```

Comment:
Please remove this auto-generated comment :)

Reply:
Done